package.json

```
@@ -10,20 +10,29 @@
     "node": "^16.6.1",
     "npm": ">=7.20.0"
   },
-  "main": "index.js",
+  "exports": {
+    "./style/standard": "./style/standard.js",
+    "./style/verbose": "./style/verbose.js",
```

Can this be compressed to:
```suggestion
    "./style": "./style",
```

---

or `"./style": "./style/"`